source/common/buffer/watermark_buffer.h

```diff
@@ -73,7 +73,7 @@ using WatermarkBufferPtr = std::unique_ptr<WatermarkBuffer>;
 class WatermarkBufferFactory : public WatermarkFactory {
 public:
   // Buffer::WatermarkFactory
-  InstancePtr create(std::function<void()> below_low_watermark,
+  InstancePtr create(BufferType /*buffer_type*/, std::function<void()> below_low_watermark,
```

Comment:
why commented out?

Reply:
Unused argument.  I removed the argument name instead.